phase1/gce/configure-vm-kubeadm.sh

```diff
@@ -27,7 +27,7 @@ fi
 
 case "${ROLE}" in
   "master")
-    kubeadm init --token "${TOKEN}" --apiserver-bind-port 443 --skip-preflight-checks --apiserver-advertise-address "$(get_metadata "k8s-advertise-addresses")" --kubernetes-version $KUBERNETES_VERSION
+    kubeadm init --token "${TOKEN}" --pod-network-cidr 10.244.0.0/16 --apiserver-bind-port 443 --skip-preflight-checks --apiserver-advertise-address "$(get_metadata "k8s-advertise-addresses")" --kubernetes-version $KUBERNETES_VERSION
```

Comment:
Sure, I'll hold off on this review for now, but you might want to proactively start incorporating the redesign. This isn't just a flag rename, but a change in behavior: instead of having different phase3 options to toggle each CNI provider on or off, it'll require switching on a single configuration value to determine which set of jsonnet files to use, which is a pattern we don't have in phase3 yet.

Also, are you planning on supporting backwards-compatibility with the old `phase3.weave_net` configuration option? Between this PR and https://github.com/kubernetes/test-infra/pull/2771, it's unclear what the full plan is to not break existing consumers of kubernetes-anywhere (and our e2e tests).